server/graphql/core/src/generic_filters.rs

```diff
@@ -46,6 +46,7 @@ impl<I: InputType> EqualFilterInput<I> {
                 .equal_any
                 .map(|inputs| inputs.into_iter().map(&to_domain).collect()),
             not_equal_all: None,
+            is_null: None,
```

Comment:
is null is required to check if integration error or integration_datetime are populated (to know which sync buffer rows to integrate), this is only really used in downstream PR, but wanted to test in repository, while it was added

Reply:
Hmm, I've made an issue for discussion: https://github.com/openmsupply/open-msupply/issues/340, I guess we need to have a quick think about pros/cons, especially in the context of 'confusion'